src/GenAPI/Microsoft.DotNet.GenAPI/AssemblySymbolOrderer.cs

```diff
@@ -0,0 +1,66 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DotNet.GenAPI
+{
+    /// <summary>
+    /// Provides ordering for namespaces, types and members.
+    /// </summary>
+    public static class AssemblySymbolOrderer
+    {
+        /// <summary>
+        /// Sorts the elements of a INamespaceSymbol.
+        /// </summary>
+        /// <param name="namespaces">List of namespaces to be sorted.</param>
+        /// <returns>Returns namespaces in sorted order.</returns>
+        public static IEnumerable<INamespaceSymbol> Order(IEnumerable<INamespaceSymbol> namespaces)
```

Comment:
These methods would be perfect candidates for extension methods. Did you consider such instead of static access via the `AssemblySymbolOrderer` class?

Reply:
Thanks. Will update it https://github.com/dotnet/sdk/pull/29471